gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/domain/SweDichotomyValidationData.java

```diff
@@ -16,22 +15,23 @@
 public class SweDichotomyValidationData {
 
     private final RaoResponse raoResponse;
-    private final AngleMonitoringResult angleMonitoringResult;
+    private final AngleMonitoringStatus angleMonitoringStatus;
 
-    public SweDichotomyValidationData(RaoResponse raoResponse, AngleMonitoringResult angleMonitoringResult) {
-        this.raoResponse = raoResponse;
-        this.angleMonitoringResult = angleMonitoringResult;
+    public enum AngleMonitoringStatus {
+        SECURE, UNSECURE, FAILURE, NONE
     }
 
-    public SweDichotomyValidationData(RaoResponse raoResponse) {
-        this(raoResponse, null);
+    public SweDichotomyValidationData(RaoResponse raoResponse,
+                                      AngleMonitoringStatus status) {
+        this.raoResponse = raoResponse;
+        this.angleMonitoringStatus = status;
     }
 
     public RaoResponse getRaoResponse() {
         return raoResponse;
     }
 
-    public AngleMonitoringResult getAngleMonitoringResult() {
-        return angleMonitoringResult;
+    public AngleMonitoringStatus getAngleMonitoringResult() {
```

Comment:
Name this method `getAngleMonitoringStatus` instead of `getAngleMonitoringResult`